### PR TITLE
Fix code quality

### PR DESCRIPTION
**Pull request by Ramanathan0908**

### CODE REVIEW

File: src/main/java/seedu/uninurse/model/task/TaskList.java [hunk 1]
```diff
@@ -157,26 +166,35 @@ public List<ListModificationPair> getDiff(GenericList<Task> otherTaskList) {
...
 
+    /**
+     * Checks whether the TaskList has the given Task.
+     *
+     * @param task to check for.
+     * @return Whether the given Task is in TaskList.
+     */
     public boolean hasTask(Task task) {
         return internalTaskList.contains(task);
     }
```
Comment (BlopApple) [inline src/main/java/seedu/uninurse/model/task/TaskList.java:200]: I think this method should be removed as we would not want to provide an explicit way to check if `TaskList` contains a certain `Task` as no other objects would need to know this information.

**Review by BlopApple — CHANGES_REQUESTED** [review]
Just some minor changes.

File: src/main/java/seedu/uninurse/logic/parser/DeleteTaskCommandParser.java [hunk 2]
```diff
@@ -1,39 +1,38 @@
...
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteTaskCommand.MESSAGE_USAGE), pe);
+            List<Index> indices = ParserUtil.parseTwoIndex(argMultimap.getPreamble());
+            return new DeleteTaskCommand(indices.get(0), indices.get(1));
+        } catch (IllegalValueException ive) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    DeleteTaskCommand.MESSAGE_USAGE), ive);
         }
-
-        return new DeleteTaskCommand(indices.get(0), indices.get(1));
```
[the remark this answers is not in the document]
Comment (Ramanathan0908) [reply src/main/java/seedu/uninurse/logic/parser/DeleteTaskCommandParser.java:37]: But parseTwoIndex() does not catch IllegalValuException, hence not having this fails some test cases
Screenshot 2022-11-05 at 10 09 31 AM

File: src/main/java/seedu/uninurse/model/task/TaskList.java (hunk 1 above)
Comment (Ramanathan0908) [reply src/main/java/seedu/uninurse/model/task/TaskList.java:200]: Sure that makes sense

File: src/main/java/seedu/uninurse/logic/parser/DeleteTaskCommandParser.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (BlopApple) [reply src/main/java/seedu/uninurse/logic/parser/DeleteTaskCommandParser.java:37]: Ok I will look into this so I'll approve for now.

**Review by BlopApple — APPROVED** [review]
LGTM

**Pull request merged**